src/main/java/com/puppycrawl/tools/checkstyle/site/XdocsTemplateSink.java

```diff
@@ -49,6 +56,40 @@ public class XdocsTemplateSink extends XdocSink {
     public XdocsTemplateSink(Writer writer, String encoding) {
         super(writer);
         this.encoding = encoding;
+        this.writer = new PrintWriter(writer);
+    }
+
+    /**
+     * Customization of org.apache.maven.doxia.sink.impl.XhtmlBaseSink.unifyEOL() method.
+     *
+     * @param text the text to scan.
+     * @return a String that contains only Linux System EOLs.
+     */
+    protected String unifyToLinuxEndOfLine(String text) {
```

Comment:
as this is method that is crafted by us, please make it private.

Reply:
Ok.

Reply:
Not done

Reply:
@kalpadiptyaroy, ping

Reply:
Sorry I was a bit occupied. Pushed in latest commit. Finally done.